### PR TITLE
I updated my idea 2&3

### PR DESCRIPTION
**Pull request by lannguyen39**

### CODE REVIEW

File: Idea 2.md
Comment (nmkwangg) [inline Idea 2.md:1]: Interesting idea. But I think that you can go beyond basic charts. While pie charts and bar charts are useful, consider using more advanced techniques like correlation analysis to link background choices to focus levels.

File: Research Idea 3.md
Comment (nmkwangg) [inline Research Idea 3.md:1]: Creative idea. You might want to broaden analysis methods. While comparing areas with and without farmers markets is good, consider using difference-in-differences or regression analysis to better isolate the farmers market’s effects from other variables.

**Comment by ernbilen** [comment]
Really great topic and presentation. One suggestion I have is, if there are any data similar to Christine's from emergency rooms and if they share anything about past smoking. There, you could work on estimating the impact on smoking. There is some work on this. For example, if there are certain states that make smoking more expensive by putting taxes, that creates a natural experiment for the effect of smoking on health. You should check out the past seminars at: https://www.tobaccopolicy.org (also scroll all the way down and look at the people who are working on maintaining it😎)

**Comment by ernbilen** [comment]
Once again, great topic + presentation. Things you can consider for your final version of your project:
- Scrape episodes and do episode level analysis.
- Scrape user comments for text analysis.
- Studio level analysis (you already have studio data which is great)
- Character level analysis- are there superstar characters that drive the anime's points singlehandedly? You can create an "all time best anime characters" list recommended by your model.
- Maybe an anime recommendation system for users? Like, if you like X, Y, Z anime, what 10 animes would your model recommend? It could go nicely in a streamlit dashboard.

**Pull request closed without merging**